lib/src/block_syntaxes/table_syntax.dart

```diff
@@ -71,27 +72,16 @@ class TableSyntax extends BlockSyntax {
   }
 
   List<String?> _parseAlignments(String line) {
-    final startIndex = _walkPastOpeningPipe(line);
+    final columns = line.replaceAll(RegExp(r'^\s*\||\|\s*$'), '').split('|');
```

Comment:
This feels a little fishy to me, like perhaps an unnecessary callout to the regular expression engine. I think it could be inefficient. It took me a minute to see that we're matching two patterns, `^\s*\|` and `\|\s*$`. I think we can instead use to RegExps like:

    static final p1 = RegExp(r'^\s*\|');
    static final p2 = RegExp(r'\|\s*$');
    // ...
    final columns = line.replaceFirst(p1, '').replaceFirst(p2, '').split('|');


Even beyond that, I'd want to make sure that calling out to the RegExp engine is not slower (or much slower) than the hand-written byte-walking which this replaces.

Reply:
Yes, I doubt the performance too. Now it should be better.